.github/workflows/pulumi-cli-dev-version.yml

```diff
@@ -0,0 +1,57 @@
+env:
+  GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
+
+name: Update latest-dev-version
+on:
+  repository_dispatch:
+    types:
+      - pulumi-cli-dev-version
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    needs: update-latest-dev-version
+    steps:
+      - name: checkout docs repo
+        uses: actions/checkout@v2
+      - name: pull-request
+        uses: repo-sync/pull-request@v2
+        with:
+          source_branch: "pulumi/${{ github.run_id }}-${{ github.run_number }}"
+          destination_branch: "master"
+          pr_title: "Regen dev version pulumi@${{ github.event.client_payload.dev_version}}"
+          pr_body: "Automated PR"
+          pr_label: "automation/pulumi-cli-docs,automation/merge"
```

Comment:
Since we haven't seen this workflow actually run yet, I'd suggest leaving auto-merge out our for the first few cycles, just so we can review these PRs before we flip the automation switch. That ok? 

```suggestion
```

Reply:
Yes, sounds like a good idea to me!